src/protocol_nmea.cpp

```diff
@@ -217,7 +217,7 @@ char *ASCII_to_f64(double *vec, char *ptr)
 	return ptr;
 }
 
-char *ASCII_to_vec4u8(uint8_t vec[], char *ptr)
+char *ASCII_to_ver4u8(uint8_t vec[], char *ptr)
```

Comment:
Did you mean to rename this, for vec4 to ver4?  I'm getting this is our version info, and not a 4-element vector?

Reply:
Yes.  I figured referring to it as a version rather than a vector is more appropriate as the version is decimal delimited were as a vector is comma delimited.